server/messages.go

```diff
@@ -141,20 +145,24 @@ func processMessages(conn net.Conn, c chan *packetInfo, t *Tracker) {
 		}
 
 		if err := scanner.Err(); err != nil {
-			fmt.Fprintf(os.Stderr, "[Error] %s\n", err.Error())
-			break
+			fmt.Fprintf(os.Stderr, "[Error] message scanner: %s\n", err)
+			return
```

Comment:
This break and another one below - I didn't see any need for them so I standardized on return

Reply:
Functionally equivalent.  I guess return is much clearer.

Dunno what the difference is between err & err.Error(). My go lore is rusty... :)

Reply:
They are equivalent @cculianu =)